### ISSUE TITLE
History of urethral procedure or catheterization

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10014]
**Preferred term label:**
History of urethral procedure or catheterization

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Peyronie disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000086
- label: History of urethral procedure or catheterization
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: 
- def: A past medical history of a procedure or catheterization involving the urethra.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10014
- synonyms:
